CMakeLists.txt

```diff
@@ -262,6 +263,7 @@ message(STATUS "SUSE 15: ${SSG_PRODUCT_SLE15}")
 message(STATUS "Ubuntu 14.04: ${SSG_PRODUCT_UBUNTU1404}")
 message(STATUS "Ubuntu 16.04: ${SSG_PRODUCT_UBUNTU1604}")
 message(STATUS "Ubuntu 18.04: ${SSG_PRODUCT_UBUNTU1804}")
+message(STATUS "McAfee VirusScan Enterprise for Linux: ${SSG_PRODUCT_VSEL}")
```

Comment:
Let's change VirusScan Enterprise for Linux to VSEL

Reply:
fixed